libs/remix-ui/run-tab/src/lib/components/contractGUI.tsx

```diff
@@ -299,8 +299,8 @@ export function ContractGUI(props: ContractGUIProps) {
           <div className="d-flex p-0 wrapperElement" onClick={handleActionClick} data-id={buttonOptions.dataId} data-title={buttonOptions.title}>
             <button
               className={`udapp_instanceButton text-nowrap overflow-hidden text-truncate ${props.widthClass} btn btn-sm ${buttonOptions.classList}`}
-              data-id={buttonOptions.dataId}
```

Comment:
Are these needed? `"firstbullseye"` and `"bullseye"`

Reply:
done